docs/guides/reward-distributor-bot.md

```diff
@@ -0,0 +1,64 @@
+# Reward distribution bot
+
+## Introduction
+
+Permissionless reward distribution bot for Lido staking modules. Operates with smart contract based on [Node Operator Registry](https://docs.lido.fi/contracts/node-operators-registry) smart contract. 
+After [Accounting Oracle](https://docs.lido.fi/guides/oracle-spec/accounting-oracle) completes the third phase, anyone can initiate the reward distribution to allocate rewards among Node Operators in the Staking Module.
+
+## Requirements
+
+### Hardware
+
+- 1-core CPU
+- 2GB RAM
+
+### Nodes
```

Comment:
Does it need a node or just an RPC endpoint? Light node, full node, archive node?

Reply:
Whatever, no special requirements